src/components/Header.tsx

```diff
@@ -1,12 +1,18 @@
-import React from 'react';
 import styled from 'styled-components';
-import { Link } from 'react-router-dom';
+import { Link, useLocation } from 'react-router-dom';
+
+const title: { [key: string]: string } = {
+  '/': '보유카드',
+  '/addCard': '카드 추가',
+};
```

Comment:
```suggestion
const title = {
  '/': '보유카드',
  '/addCard': '카드 추가',
};
```
객체 리터럴과 index signature의 차이도 찾아보시면 좋겠습니다~
타입읂 항상 좁혀서 써야한다는 것두요!